job/common/src/main/java/alluxio/job/plan/transform/CompactConfig.java

```diff
@@ -161,4 +163,9 @@ public String toString() {
   public String getName() {
     return NAME;
   }
+
+  @Override
+  public Collection<String> affectedPaths() {
+    return ImmutableList.of(mInput, mOutput);
```

Comment:
curious why affectedPath includes input?

Reply:
it may be a bit weird but I think including input is correct from the user's point of view.